SampleProject/Models/VIThing.m

```diff
@@ -3,7 +3,7 @@
 //  SampleProject
 //
 //  Created by Sean Wolter on 8/23/14.
-//
+//  Copyright © 2015 Vokal.
 //
 
 #import "VIThing.h"
```

Comment:
For the major-version-bump release, would it be reasonable to refactor all the `VI` classes to be `VOK` classes?